zerver/models.py

```diff
@@ -282,7 +282,7 @@ def get_realm(string_id):
     return Realm.objects.filter(string_id=string_id).first()
 
 def completely_open(realm):
-    # type: (Realm) -> bool
+    # type: (Optional[Realm]) -> bool
```

Comment:
Let's fix the style of the check just under this to `is not None`, while we're fixing `None`-related issues in this bit of code.